generators/form/templates/test-data.json.ejs

```diff
@@ -0,0 +1,4 @@
+{
+  "data": {
+  }
+}
```

Comment:
Unsure if there are any meaningful defaults for this or if there's anything that could be generated from the generator choices.

I left it empty since it's likely going to be edited or replaced by the developer anyway.